### PR TITLE
mgr/cephadm: document pool requirement for NFS-Ganesha service

### PR DESCRIPTION
**Pull request by bk201**
Add a note in the document to remind users.

Signed-off-by: Kiefer Chang <kiefer.chang@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by bk201** [comment]
> You have missed adding doc prefix in commit title. Please change it to `doc/cephadm: Add pool requirement note for NFS-Ganesha service`. Otherwise looks good.

Thanks! Fixed.

**Comment by bk201** [comment]
jenkins render docs

File: doc/cephadm/install.rst [hunk 1]
```diff
@@ -367,6 +367,9 @@ RADOS pool *nfs-ganesha* and namespace *nfs-ns*,::
 
   # ceph orch apply nfs foo nfs-ganesha nfs-ns
 
+.. note::
+   Please create pool *nfs-ganesha* first if it doesn't exist.
```
Comment (asettle) [inline doc/cephadm/install.rst:371]:
```suggestion
   Create the *nfs-ganesha* pool first if it doesn't exist.

Comment (bk201) [reply doc/cephadm/install.rst]: Really appreciate! Fixed.

**Pull request merged**